source/driver/Driver.cpp

```diff
@@ -346,6 +346,10 @@ bool Driver::processOptions() {
     diagEngine.setErrorLimit((int)options.errorLimit.value_or(20));
     diagEngine.setDefaultWarnings();
 
+    // Demoted to warning just to get the -W flag options, but we really
```

Comment:
Can you mention that this is to support compatibility with other tools? Something like "Some tools allow ignoring duplicate module/interface/program definitions, so this is a suppressible warning that we promote to an error by default."